html/core/src/jsMain/kotlin/org/jetbrains/compose/web/dom/Base.kt

```diff
@@ -67,8 +67,9 @@ private class DomElementWrapper(override val node: Element): DomNodeWrapper(node
 
                 val style = node.unsafeCast<ElementCSSInlineStyle>().style
 
-                styleApplier.properties.forEach { (name, value) ->
-                    style.setProperty(name, value.toString())
+                styleApplier.properties.forEach { (name, value, important) ->
+                    if (important) error("important")
```

Comment:
why does it throw an error when `important` is true?

Reply:
Fixed